### ISSUE TITLE
CLI usage with npx (Windows)

### ISSUE THREAD

**Issue by piotr-cz** [eslint/eslint#13733]
**Tell us about your environment**

Windows 10

* Node version: v14.13.0
* npm version: v6.14.8
* Local ESLint version: v7.10.0
* Global ESLint version: Not found

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?**
default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```json
{
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
// Not related
```

_Case A_
```bash
npx eslint --rule "quotes: [2, double]" src
```

_Case B_
```bash
npx eslint -- --rule "quotes: [2, double]" src
```

This works:

```bash
node_modules\.bin\eslint --rule "quotes: [2, double]" src
```

**What did you expect to happen?**
Command should show report for each file in `src` directory, as in [docs](https://eslint.org/docs/user-guide/command-line-interface#-rule)

**What actually happened? Please include the actual, raw output from ESLint.**

_Case A_
```
Oops! Something went wrong! :(

ESLint: 7.10.0

No files matching the pattern "[2," were found.
Please check for typing mistakes in the pattern.
```

_Case B_
```
Oops! Something went wrong! :(

ESLint: 7.10.0

No files matching the pattern "--rule" were found.
Please check for typing mistakes in the pattern.
```

**Are you willing to submit a pull request to fix this bug?**
no

**Comment by mdjermanovic** [comment]
Hi @piotr-cz, thanks for the issue!

This seems to be a bug in npx: https://github.com/npm/npx/issues/43, https://github.com/npm/npx/issues/10

It works without spaces in quotes:

```bash
npx eslint --rule "quotes:[2,double]" src
```

**Comment by piotr-cz** [comment]
@mdjermanovic 
Thank you for looking it up instead of me, it's clearly a `npx` issue.
Your workaround works 👍
